bin/benchmark-run-all.sh

```diff
@@ -139,22 +144,22 @@ do
     export CONFIG
     export OUTPUT_FOLDER
 
-    if [ -n "$RESTART_SERVERS" ]; then
+    if [[ "$RESTART_SERVERS" == "true" || "$RESTART_SERVERS" != "" ]] && [[ "$RESTART_SERVERS" != "false" ]]; then
         /bin/bash ${SCRIPT_DIR}/benchmark-servers-start.sh ${CONFIG_INCLUDE}
 
         sleep 3s
     fi
 
     /bin/bash ${SCRIPT_DIR}/benchmark-drivers-start.sh ${CONFIG_INCLUDE}
 
-    if [ -n "$RESTART_SERVERS" ]; then
+    if [[ "$RESTART_SERVERS" == "true" ]]; then
```

Comment:
```suggestion
    if [[ "$RESTART_SERVERS" != "false" ]]; then
```
We should stop servers here if `RESTART_SERVERS` is either `"true"` or `"<IP>:<delay>:<pause>:<period>"`